.gitignore

```diff
@@ -0,0 +1,9 @@
+# For now, as extensions are in dev mode
+/composer.lock
+/var/*
+!/var/sqlite/
+/vendor
+/public/*
```

Comment:
I propose sorting the `.gitignore` entries by name.

Reply:
I've group sorted. Some parts are excluded and would be sorted separate as they are prefixed with `!`.